### ISSUE TITLE
Broken ZeroHello site

### ISSUE THREAD

**Issue by filips123** [HelloZeroNet/ZeroHello#116]
ZeroHello site is broken. Tested in Firefox, Chrome and Edge with fresh-installed ZeroNet 0.6.5.

![screenshot](https://user-images.githubusercontent.com/16626308/55689230-52209b80-5982-11e9-8b76-137fc2c81923.png)

If I open browser console in Firefox, I see a lot of Content Security Policy errors.

But main problem is that http://127.0.0.1:43110/1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D/js/all.js is not found. It is same in all other browsers.

**Comment by HelloZeroNet** [comment]
I just tested on a new installation and it's works for me. 
Do you have all.js on your hdd data/1HeLLo.../js directory?
If you open log/debug.log is there a line that contains "Verify correct: js/all.js" (or any other reference to js/all.js?)
Whats the value of the "modified" node in the file "data/1Hello../content.json" ? (it should be 1553514828)

**Comment by filips123** [comment]
I think I know what's the problem.

I downloaded ZeroNet to `Program Files` on Windows because I want to use it globally. And this is probably the problem because `ZeroNet.exe` can't write to that folder.

Is it possible to fix this? If possible, I want to have `log` and `data` directories in `%APPDATA%\ZeroNet`.

**Comment by HelloZeroNet** [comment]
You can change by putting a zeronet.conf next to zeronet.exe:

```
[global]
log_dir = C:\ProgramData\ZeroNet\log
data_dir = C:\ProgramData\ZeroNet\data
```
This will put the data to c:\Users\[username]\AppData\Local\ZeroNet\data
The %APPDATA% and other variables not supported yet.

**Comment by filips123** [comment]
I fixed this by adding the following lines to `Config.py` that check if the program is running from `Program Files` and then set `start_dir` to `%APPDATA\ZeroNet`.

```py
        elif this_file.startswith("C:/Program Files"):
            # Running in non-writeable directory, put data to AppData
            start_dir = os.path.expandvars("%APPDATA%/ZeroNet").replace("\\", "/")
```

Is that OK? Or should I use `%LOCALAPPDATA%`?
If it is, I can submit a pull request.

**Comment by HelloZeroNet** [comment]
Yes, it's looks good for me. I think %LOCALAPPDATA% is the proper place for it. (does not travels on the network with the user)

**Comment by filips123** [comment]
I changed `start_dir` but there are still some problems.

Tor doesn't work. I get this error in ZeroHello:

```
Status: Error ([Errno 10061] [Error 10061] No connection could be made because the target machine actively refused it.)
```

After quitting ZeroNet, processes still exist (`ZeroNet.exe`, `meek-client.exe`). I have to manually kill them from Task Manager.

These problems are probably because of non-writable directory because all things work if I run ZeroNet from the writable directory.

Can you make some installer for ZeroNet that will correctly set up all directories?

**Comment by filips123** [comment]
I created https://github.com/HelloZeroNet/ZeroNet/pull/1972. Should I also submit it to `py3` branch?
